intellij-ext/src/main/resources/META-INF/mirrord-js.xml

```diff
@@ -2,4 +2,9 @@
     <extensions defaultExtensionNs="JavaScript">
         <nodeRunConfigurationExtension implementation="com.metalbear.mirrord.products.nodejs.NodeRunConfigurationExtension" id="mirrordJS"/>
     </extensions>
+
+    <projectListeners>
+        <listener class="com.metalbear.mirrord.MirrordNpmExecutionListener"
```

Comment:
why not add it in the general `plugin.xml`? I'm not sure if that's okay to add the same name in two different, also this is quite agnostic to IDE in terms of dependencies, no?

Reply:
On second thought maybe limiting to where I saw the presence of `com.intellij.lang.javascript.buildTools.npm.rc.NpmRunConfiguration` doesn't quite work like I thought, and yes dependencies are completely agnostic to what IDE loads them